move_together_app/lib/core/services/api_services.dart

```diff
@@ -243,4 +245,34 @@ class ApiServices {
       throw Exception('Failed to get transport');
     }
   }
+
+  Future<Trip> createTrip(Trip trip) async {
+    String formattedStartDate = DateFormat("yyyy-MM-ddTHH:mm:ss'Z'").format(trip.startDate.toUtc());
+    String formattedEndDate = DateFormat("yyyy-MM-ddTHH:mm:ss'Z'").format(trip.endDate.toUtc());
```

Comment:
```suggestion
    String formattedStartDate = trip.startDate.toIso8601String();
    String formattedEndDate = trip.endDate.toIso8601String();
```

Passe les dates en ISO 8601